package.json

```diff
@@ -14,9 +14,9 @@
     "clean": "pnpm clean:modules && pnpm clean:builds",
     "clean:builds": "find . \\( -name \"dist\" -o -name \".next\" \\) -type d -prune -exec rm -rf '{}' +",
     "clean:modules": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
-    "dev": "pnpm docker:compose:legacy up -d --build && pnpm wallet-frontend dev",
+    "dev:legacy": "pnpm docker:compose:legacy up -d --build && pnpm wallet-frontend dev",
     "prod": "pnpm docker:compose:prod up -d --build",
-    "dev:update": "pnpm docker:compose up -d --build && pnpm wallet-frontend dev",
+    "dev": "pnpm docker:compose up -d --build && pnpm wallet-frontend dev",
```

Comment:
Will update and remove scripts in a separate PR. For now `pnpm dev` will run the wallet application with the new backend.